GraphConsoleAppV3/GraphRequests.cs

```diff
@@ -0,0 +1,1551 @@
+        {
+            #region Update User properties
+
+            //*******************************************************************************************
+            // update the newly created user's Password, PasswordPolicies and City
+            //*********************************************************************************************
+            if (newUser.ObjectId != null)
+            {
+                // update User's info
+                newUser.City = "Seattle";
+                newUser.Country = "UK";
+                newUser.Mobile = "+4477889456789";
+                newUser.UserType = "Member";
+
+                try
+                {
+                    await newUser.UpdateAsync();
+                    Console.WriteLine("\nUser {0} was updated:", newUser.DisplayName);
+                    Console.WriteLine("\t{0}:\t{1}", "City", "Seattle");
+                    Console.WriteLine("\t{0}:\t{1}", "Country", "UK");
+                    Console.WriteLine("\t{0}:\t{1}", "Mobile", "+4477889456789");
+                    Console.WriteLine("\t{0}:\t{1}", "Type", "Member");
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError Updating the user {0}", Program.ExtractErrorMessage(e));
+                }
+            }
+
+            #endregion
+        }
+
+        private static async Task ResetUserPassword(IUser newUser)
+        {
+            #region Reset password for user
+
+            //*******************************************************************************************
+            // update the newly created user's Password and PasswordPolicies
+            // requires Directory.AccessAsUser.All and that the current user is a user, helpdesk or company admin
+            //*********************************************************************************************
+            if (newUser.ObjectId != null)
+            {
+                // update User's password policy and reset password - forcing change password at next logon
+                PasswordProfile PasswordProfile = new PasswordProfile
+                {
+                    Password = "changeMe!",
+                    ForceChangePasswordNextLogin = true
+                };
+                newUser.PasswordProfile = PasswordProfile;
+                newUser.PasswordPolicies = "DisablePasswordExpiration, DisableStrongPassword";
+                try
+                {
+                    await newUser.UpdateAsync();
+                    Console.WriteLine("\nUser {0} password and policy was reset", newUser.DisplayName);
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError Updating the user {0}", Program.ExtractErrorMessage(e));
+                }
+            }
+
+            #endregion
+        }
+
+        private static async Task AssignManager(IUser newUser,

```

Comment:
done

---
In reply to: [81251091](https://github.com/Azure-Samples/active-directory-dotnet-graphapi-console/pull/37#discussion_r81251091) [](ancestors = 81251091)